frontend/ui/lib/widgets/place_block_widget.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+class PlaceBlockWidget extends StatefulWidget {
+  final String placeName;
```

Comment:
No need to update this here, but you'll likely want to take in a place model object instead of a String later.